eta/core/datasets.py

```diff
@@ -1943,11 +2000,12 @@ def _to_helper_list_video_objects(self, records):
             NO_ID = 'NO_ID'
             helper_dict = defaultdict(set)
 
-            for frame_no, frame in labels.frames.items():
+            for frame_no in labels:
                 if (frame_no < record.clip_start_frame
                         or frame_no >= record.clip_end_frame):
                     continue
 
+                frame = labels[frame_no]
```

Comment:
is this an optimization? why was this changed?

Reply:
Syntax cleanup.